.github/workflows/test-portable.yaml

```diff
@@ -0,0 +1,24 @@
+name: Portable Test Sigstore Containers
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-latest
+          # - macos-latest # docker is not available in github's hosted macos runners.
+          # - windows-latest # hold off while rekor still specifies `platform: linux`
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+      - uses: ./actions/portable_testing
+      - name: healthcheck
+        run: |
+          curl http://localhost:5555/healthz
+          curl http://localhost:3000/
```

Comment:
this could really use a comment (or maybe just a more descriptive name)

Reply:
It seemed like it was faster, but I can see now that the caching might not actually be happening. I'm trying fix this.

- https://github.com/sigstore/scaffolding/actions/runs/13926652632/job/38972863790#step:7:3

```
Warning: Path Validation Error: Path(s) specified in the action for caching do(es) not exist, hence no cache is being saved.
```

Reply:
Caching works now, but it's somehow slower than not using the cache, so i'm disabling it, and users can enable it with an option.